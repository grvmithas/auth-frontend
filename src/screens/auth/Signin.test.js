import React from 'react'
import SignIn from './Signin'

test('renders signin', () => {
  expect(SignIn).toMatchSnapshot()
})

