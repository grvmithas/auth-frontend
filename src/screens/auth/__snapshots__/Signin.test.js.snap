// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders signin 1`] = `[Function]`;
